ref/Boolean_Operators.md

```diff
@@ -181,7 +181,14 @@ This operator is similar to **&gt;** but tests for **less than or equal to**.
 
 ------
 
-#####  Fuzzy comparisons: `~=`, `~!=`, `~<`, `~>`, `~>=`, `~<=`
+####  Fuzzy comparisons:
+
+##### Approximately equal: `‹expr1› ~= ‹expr2›`
+##### Not approximately equal: `‹expr1› ~!= ‹expr2›`
```

Comment:
> http://doc.cinderella.de/tiki-index.php?page=Operator+Index under 'Infix operators'

Ah, thanks. I didn't fetch that since I intend to auto-generate the index eventually.

> And I would really stick with 'approximately not equal', etc.

I had a look at Unicode. It calls ≉ [“not almost equal to”](http://www.charbase.com/2249-unicode-not-almost-equal-to), ⪅ [“less-than or approximate”](http://www.charbase.com/2a85-unicode-less-than-or-approximate) and ⪉ [“less-than and not approximate”](http://www.charbase.com/2a89-unicode-less-than-and-not-approximate). I think these terms are closer to my formulation than the original one. I guess someone invested some effort into these names, but I'm not sure. I also [asked about this on Math SE](http://math.stackexchange.com/q/1662241/35416) and would like to wait a bit to see what answers come in there.

> The picture from p. 254 of the printed book might be helpful as well.

I don't have the book at hand just now, but I imagine you're referring to [this figure](https://github.com/CindyJS/CindyJS/blob/6662cc2e0a902d3be2ea590a3ef23545a32d494c/ref/img/Comparisons.png) which is already included in that document. It's indeed massively useful, and up until recently I had to refer to that in order to get the distinction between `~<` and `~<=` right. Now with my new names and the association to ⪉ and ⪅ set in my mind, I think I finally will remember them. Our discussion here surely helps in that respect, too.